sisl/viz/inputFields/color.py

```diff
@@ -0,0 +1,9 @@
+from ..inputField import InputField
+
+class ColorPicker(InputField):
+    
+    _type = 'color'
+    
+    _default = {
+        "width": "s50% m30% l15%"
+    }
```

Comment:
newline (perhaps)

Reply:
@pfebrer96 also, no camel-case in modules. If the sub-module is meant to be hidden, then prefix with `_input_fields`